### PR TITLE
Align transport and package naming convention

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: src/Transport/AzureStorageQueueInfrastructure.cs [hunk 1]
```diff
@@ -172,11 +172,9 @@ AzureStorageAddressingSettings GetAddressing(ReadOnlySettings settings, IProvide
                 accounts = new AccountConfigurations();
             }
 
-            const string defaultAccountAlias = "";
-
             addressing.SetAddressGenerator(addressGenerator);
-            addressing.RegisterMapping(accounts.defaultAlias ?? defaultAccountAlias, accounts.mappings);
-            addressing.Add(new AccountInfo(defaultAccountAlias, queueServiceClientProvider.Client), false);
+            addressing.RegisterMapping(accounts.defaultAlias, accounts.mappings);
```
Comment (SeanFeldman) [inline src/Transport/AzureStorageQueueInfrastructure.cs:176]: Are you sure about this one?

**Review by SeanFeldman — APPROVED** [review]
Approving but that default alias... I saw that null causing an issue when exchanging message with an older endpoint.

**Comment by danielmarbach** [comment]
Yeah that slipped through. Already fixed

**Comment by danielmarbach** [comment]
`MessageWrapper` has still `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` because I wasn't sure if it can be moved. Did we ever serialize the type info with ASQ? We probably didn't right? If that is true we could technically move it

File: src/Transport/AzureStorageQueueInfrastructure.cs (hunk 1 above)
Comment (danielmarbach) [reply src/Transport/AzureStorageQueueInfrastructure.cs]: Already force pushed and updated

**Pull request merged**